projects/biometrics/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: page
+title: Biometric Privacy
+image: images/biometrics/titleimage.jpg
+permalink: /projects/biometrics/
+---
+
+![protest](/images/biometrics/protest.jpg)
+---
+---
+---
+---
+
+Lucy Parsons Labs (LPL) is committed to protecting the civil liberties of
+Chicagoans and the people of Illinois by shining light on the dangers of
+biometric surveillance and facial recognition technology, which should be
+banned immediately.  
+
+Emerging technologies such as facial recognition, biometric identification,
+social media monitoring, license plate monitoring, and predictive policing, are
+often used as a panacea for disrupting violence and securing our communities.
+Police officers claim that these technologies help resolve crime, prevent
+terrorism, and locate missing children. However, as the Chicago Police
+Department continues to [spy on the
+victims](https://onezero.medium.com/chicago-cops-use-social-media-to-track-grieving-families-of-gunshot-victims-e68e5a6dc40c)
+of violent crimes, they solved less than 16 percent of murders during the first
+half of 2018.
+
+Since our founding in October 2015, this Chicago-based non-profit has filed
```

Comment:
Suggestion:

`Since our founding in Chicago in October 2015, Lucy Parsons Labs has filed`